make/platform/Makefile.linux64

```diff
@@ -17,3 +17,5 @@
 
 # Makefile.linux64
 # Special AR and RANLIB definitions for XMT used to be here.
+
+CXXFLAGS += -frtti
```

Comment:
There are problems mixing RTTI and no-RTTI code - and LLVM wil build with no-RTTI.